UM/Qt/Bindings/MainWindow.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, parent = None):
         self._preferences.addPreference("general/window_top", 50)
         self._preferences.addPreference("general/window_state", Qt.WindowNoState)
 
+        self._preferences.addPreference("general/sidebar_collaps", False)
```

Comment:
- The preference should be in Cura, not Uranium
- The correct spelling is "sidebar_collapse"

Reply:
It also has nothing to do with the main window. It should be in CuraApplication.